requirements.txt

```diff
@@ -17,3 +17,4 @@ natsort==7.0.1
 tabulate==0.8.6
 packaging==20.4
 sqlparse==0.3.1
+asyncpg==0.21.0
```

Comment:
Is this single addition enough? How can we be sure? 
I am thinking for example to contextlib , used in controller.py

Reply:
We can be sure by testing it with a clean virtualenv. Since contextlib is part of Python, there is nothing to add here for it.